blog/breakpoint.ftd

```diff
@@ -9,74 +9,60 @@ published-on: November 3, 2023 at 3:40 pm
 post-url: /breakpoint/
 author: $authors.rithik
 
-Browser breakpoint widths are used in web design to define how a
-webpage should be displayed and styled based on the width of the
-user's browser window or device screen. These breakpoints help create
-responsive web designs that adapt to different screen sizes and
-orientations, making websites more accessible and user-friendly
-on a variety of devices, such as desktop computers, smartphones etc.
+Today, in this blog we will see how fastn allows the use of user-defined
+custom breakpoints. Although currently we can only modify the
+default breakpoint when using ftd.document. We will cover the mentioned
```

Comment:
Use `backticks` when using technical terms. Also link to `ftd.document` documentation.